Volume/Views/BrowserView.swift

```diff
@@ -12,10 +12,13 @@ import WebKit
 
 struct BrowserView: View {
     @EnvironmentObject private var userData: UserData
+    @State var shoutoutsButtonEnabled: Bool = true
```

Comment:
I would call this `isShoutoutsButtonEnabled` a la `isEnabled` property in UIKit. 

Also, I think this should be a computer var instead. See my comment below.